src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer machine started its work");
+    }
+    @Override
+    public void stopWork() {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```